app/komponenter/innholdkonteiner/innholdKonteiner.module.css

```diff
@@ -13,7 +13,7 @@
   gap: 5rem;
   align-items: center;
 
-  @media (min-width: 768px) {
+  @media (min-width: 37.5rem) {
```

Comment:
Er det noen som har tipset dere om å bruke rem her? Synes selv at det gir mest mening å bruke `px` for akkurat breakpoints når det kommer til responsivt design - siden 480px, 768px, 1024px, og 1280px har blitt ganske så standardiserte knekkpunkter over årene.

Reply:
Det gir kanskje ikkje meining å ha det i brekkepunkter. Kristine fortalte oss at man burde bruke `rem` når det kom på styling, men det er kanskje bredde og faktiske størrelser som har no å si for komponenter, og ikkje brekkepunkter? Ergo, det skal stå i piksler i media queries? Veldig usikker der.

Reply:
**Generelt for styling:** Bruk rem. Rem sin størrelse kan endre seg med font-sizen som blir satt i html-taggen til nettsiden du befinner deg i, font-sizen kan også endres av nettleserens innstillinger. Så hvis font-sizen endres skal i teorien all spacing mellom komponenter skalere bra hvis du bruker rem.

**Spesielt for setting av knekkpunkter for media-queries:** Bruk px. En piksel er en absolutt verdi. Når du setter at noe skal endre seg i en media-query ved 768px, så vil den alltid gjøre det. Hvis du setter at noe skal endre seg ved 37.5rem, så risikerer du at dette ikke skjer ved 768px på grunn av en innstilling i nettleseren som brukes eller lignende.

For å lese mer kan man for eksempel google bruksområdene til rem vs. bruksområdene til px

Reply:
Godt svar! Kristine var også enig i den. Byttet tilbake i commit a9d2600